pkg/sdk/drop.go

```
@@ -7,8 +7,8 @@ import (
 
 // SafeDrop is a helper function that wraps a drop function and handles common error cases that
 // relate to missing high hierarchy objects when dropping lower ones like schemas, tables, views, etc.
-// Whenever an object is missing or the higher hierarchy object is not accessible, it will return ErrObjectNotFound error,
-// which can be leveraged with [errors.Is] to handle the logic in case of missing objects.
+// Whenever an object is missing or the higher hierarchy object is not accessible, it will return nil error,
```

Nit: `it will not return an error`.

And `which means the caller can safely ignore the error and proceed` sounds a bit confusing because now, the error handling is "default" by the caller, (certainly not **ignored** by the caller), right? The point of this function is to not return an error when objects higher in hierarchy don't exist as well, and IMO this comment does not reflect this enough.